ui/css/logviewer.css

```diff
@@ -156,15 +157,10 @@ body {
 }
 
 .logviewer-actionbtn span {
-    color: #9fa3a5;
+    color: #777;
```

Comment:
I think you can just remove this property entirely. Testing in the devtools, I get the same result from doing that.